src/Files.App/Services/Windows/WindowsWallpaperService.cs

```diff
@@ -18,28 +18,32 @@ public sealed class WindowsWallpaperService : IWindowsWallpaperService
 		public unsafe void SetDesktopWallpaper(string szPath)
 		{
 			using ComPtr<IDesktopWallpaper> pDesktopWallpaper = default;
-			var desktopWallpaperIid = typeof(IDesktopWallpaper).GUID;
-			var desktopWallpaperInstanceIid = typeof(DesktopWallpaper).GUID;
+			var CLSID_DesktopWallpaper = typeof(DesktopWallpaper).GUID;
+			var IID_DesktopWallpaper = typeof(IDesktopWallpaper).GUID;
```

Comment:
@yaira2 we really need RVA Guid gen. FYI @hez2010

You have to put Guid variables into the local variable field first and do `&guid` as above. Plus, besides this IID's and CLSID's Guid issue, what CsWin32 generates for other Guid variables such as BHID are also not on RVA field.
This inconvenience persists everywhere we use COM generated by CsWin32.

I'm probably going to request this in CsWin32 tho, I'd like us to have SG for this as a workaround for now after we adopted .NET 9 (If the author of CsWin32 was quick to fix this, we would go with it ofc tho). I think [this SG](https://github.com/dongle-the-gadget/GuidRVAGen) does exactly what we want:

```c#
// In Files.App.CsWin32/CLSID.cs
public static unsafe partial class CLSID
{
    [RVAGuidGen("00000000-0000-0000-0000-000000000000")]
    public partial Guid* CLSID_DesktopWallpaper { get; }
}
```

The SG generates:
```c#
public static unsafe partial class CLSID
{
    public partial Guid* CLSID_DesktopWallpaper // I guess inlining too
        => (Guid*)Unsafe.AsPointer(ref MemoryMarshal.AsRef([0x00, 0x00, ...]));
}
```

---

Once CsWin32 generates Guid from ROS\<byte\>, we would create an extension method like this:

```c#
// Files.App.CsWin32/Extras.cs

public unsafe Guid* AsPointer(this iid)
{
    // NOTE: This works only with Guid variables stored in ".rodata".
    return (Guid*)Unsafe.AsPointer(ref Unsafe.AsRef(in iid));
}
```